src/scss/2021/tools/_type.scss

```diff
@@ -34,7 +34,7 @@
 	font-family: var(--font-mono);
 	font-size: var(--type-size-body-lg);
 	font-weight: weight(light);
-	line-height: 1.4;
+	line-height: 1.5rem;
```

Comment:
`line-height` ideally should not have units, it’s based on a relative ratio to the font-size.